requirements/base.txt

```diff
@@ -24,6 +25,7 @@ edgegrid-python==1.0.10
 elasticsearch==2.4.1
 feedparser==5.2.1
 geocoder==1.12.0
+github3.py==0.9.6
```

Comment:
Thinking about this a little bit more and looking at our http-smoke-tests, we currently manually run this in that Jenkins job: 

```shell
pip install requests==2.7.0 beautifulsoup4==4.5.1
```

How about instead of these in base.txt, we add requests, beautifulsoup4, boto3, and github3.py to a `requirements/automations.txt` file that we then use in Jenkins jobs that use scripts in this repo?